tests/auto-jenkins/jobs/bl.py

```diff
@@ -38,6 +38,11 @@ def set_directories(job_obj):
         blstore = '/work/noaa/epic/UFS-WM_RT/NEMSfv3gfs'
         rtbldir = '/work/noaa/stmp/role-epic-ps/stmp/role-epic-ps/FV3_RT/'\
                  f'REGRESSION_TEST'
+    elif job_obj.machine == 'hercules':
+        workdir = '/work/noaa/epic/role-epic/autort/tests/auto/pr'
+        blstore = '/work2/noaa/epic/RT/NEMSfv3gfs'
```

Comment:
@zach1221 @ulmononian should hercules blstore be /work/noaa/epic/hercules/UFS-WM_RT ?

Reply:
Yes, it should be.

Reply:
done.